### PR TITLE
add release note for 0.5

### PR DESCRIPTION
**Pull request by FMX**

### CODE REVIEW

File: docs/community/release_notes/release_note_0.5.0.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+  limitations under the License.
+---
+
+# Apache Celeborn™ 0.5.0 Release Notes
+
+## Highlight
+- Celeborn support TLS for communication
+- Memory usage and performance optimization
+- Enhanced metrics and HTTP APIs
+- Support Spark3.5 with JDK21
```
Comment (pan3793) [inline docs/community/release_notes/release_note_0.5.0.md]:
```suggestion
- Support Spark 3.5 with JDK 21
```

File: docs/community/release_notes/release_note_0.5.0.md [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+# Apache Celeborn™ 0.5.0 Release Notes
+
+## Highlight
+- Celeborn support TLS for communication
```
Comment (pan3793) [inline docs/community/release_notes/release_note_0.5.0.md]:
```suggestion
- Support TLS for internal communication
```

File: docs/community/release_notes/release_note_0.5.0.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+Thanks to the following contributors who helped to review and commit to Apache Celeborn 0.4.1 version:
+
+| Contributors |                 |                   |                 |                     |               |
+|--------------|-----------------|-------------------|-----------------|---------------------|---------------|
+| albin3       | AngersZhuuuu    | Aravind Patnam    | binjie yang     | cfmcgrady           | ChenYi015     |
+| CodingCat    | Curtis Howard   | cxzl25            | dev-lpq         | ErikFang            | FMX           |
+| ForVic       | huangxiaopingRD | jiang13021        | jiaoqingbo      | kerwin-zk           | labbomb       |
+| leixm        | lifulong        | lyy-pineapple     | miyuesc         | Mridul Muralidharan | mridulm       |
+| onebox-li    | otterc          | pan3793           | radeity         | s0nskar             | SteNicholas   |
+| tiny-dust    |   turboFei      |      waitinfuture |       wxplovecc |   xianminglei       |    xinyuwang1 |
```
Comment (pan3793) [inline docs/community/release_notes/release_note_0.5.0.md]: lint

File: docs/community/release_notes/release_note_0.5.0.md (hunk 1 above)
Comment (pan3793) [reply docs/community/release_notes/release_note_0.5.0.md]: actually, this is a little bit contrived, Spark 3.5 does not officially support Java 21

File: docs/community/release_notes/release_note_0.5.0.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+- [CELEBORN-1281] Bump Spark from 3.5.0 to 3.5.1
+- [CELEBORN-1262] Bump Spark from 3.3.3 to 3.3.4
+- [CELEBORN-1243] Bump Flink from 1.18.0 to 1.18.1
+- [CELEBORN-1221] Bump Flink from 1.17.0 to 1.17.2
+- [MINOR] Bump minikube and kubernetes version of integration test
+- [BUILD] Bump netty version to latest 4.1.107.Final
+
+## Credits
+
+Thanks to the following contributors who helped to review and commit to Apache Celeborn 0.4.1 version:
```
Comment (cfmcgrady) [inline docs/community/release_notes/release_note_0.5.0.md]:
```suggestion
Thanks to the following contributors who helped to review and commit to Apache Celeborn 0.5.0 version:
```

File: docs/community/release_notes/release_note_0.5.0.md (hunk 1 above)
Comment (pan3793) [reply docs/community/release_notes/release_note_0.5.0.md]: Maybe just "Support Java 21"?

File: docs/community/release_notes/release_note_0.5.0.md (hunk 3 above)
Comment (FMX) [reply docs/community/release_notes/release_note_0.5.0.md]: Fixed

**Comment by FMX** [comment]
Thanks @pan3793 @cfmcgrady . Thoes docs are update. PTAL.

**Review by pan3793 — APPROVED** [review]

**Review by cfmcgrady — APPROVED** [review]

**Review by RexXiong — APPROVED** [review]

**Pull request merged**